### DISCUSSION TITLE
Mongo DB Connection via VPC as Integration

### DISCUSSION THREAD

**Discussion by wortkotze** [vercel/vercel#5665] [Help] [answered]
In the Marketplace there is a mongodb Integration from atlas what is great, but the connection is nothing else then adding env variables. Would it be possible to automatically peer via a VPC in a Pro Account.

It's one of our blockers at the moment to use everything in production.
we love the platform and it would be a great benefit for us to have here an automation

thx

**Comment by leerob** [comment, marked as answer]
Hello! I wanted to give an update here. VPC peering is now on our roadmap for enterprise. Feel free to [contact us](https://vercel.com/contact/sales) if this is something your team needs. Thank you 👍
